sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/EmbeddingItem.java

```diff
@@ -19,15 +20,37 @@
 @Immutable
 public final class EmbeddingItem implements JsonSerializable<EmbeddingItem> {
 
+    /*
+     * List of embeddings value for the input prompt. These represent a measurement of the
+     * vector-based relatedness of the provided input.
+     */
+    @Generated
+    private final List<Double> embedding;
+
     private final String embeddingBase64;
 
+    /**
+     * Creates an instance of EmbeddingItem class.
+     *
+     * @param embeddingBase64 the embedding value to set.
+     * @param promptIndex the promptIndex value to set.
+     */
+    private EmbeddingItem(String embeddingBase64, List<Double> embedding, int promptIndex) {
```

Comment:
Do we need this private constructor?

Reply:
A new private constructor will be generated if we remove this private constructor. It is for passing the typespec validation.

Reply:
![image](https://github.com/user-attachments/assets/8507fafc-506f-4ed5-8c2b-c89791dd4e5b)